starfish/imagestack/indexing_utils.py

```diff
@@ -26,8 +27,42 @@ def convert_to_selector(
     return return_dict
 
 
-def index_keep_dimensions(
-        data: xr.DataArray, indexers: Mapping[str, Union[int, slice]]) -> xr.DataArray:
+def convert_coords_to_indices(array: xr.DataArray,
+                              indexers: Mapping[Coordinates, Union[Number, Tuple[Number, Number]]]
+                              ) -> Dict[Axes, Union[int, tuple]]:
+    """
+    Convert mapping of physical coordinates to value or range to mapping of corresponding Axes and
+    positional coordinates.
+
+    Parameters
+    ----------
+    array: xr.DataArray
```

Comment:
nit: @ambrosejcarr always wrote the parameters docs with a space before and after the colon, so I just copied him.